public/components/trace_analytics/components/common/search_bar.tsx

```diff
@@ -86,34 +91,30 @@ export const SearchBar = forwardRef((props: SearchBarOwnProps, ref) => {
             />
           </EuiFlexItem>
         )}
-        <EuiFlexItem grow={false} style={{ maxWidth: '40vw' }}>
-          {renderDatePicker(props.startTime, props.setStartTime, props.endTime, props.setEndTime)}
+        <EuiFlexItem grow={false} style={{ maxWidth: '30vw' }}>
```

Comment:
Nit: just for my knowledge why we need the `maxWidth` change here.

Reply:
This change is to support the new ui giving precedence to the search bar over the date picker.
At 40vw it looks like this on small screen:
<img width="833" alt="Screenshot 2024-08-23 at 12 34 58 PM" src="https://github.com/user-attachments/assets/4483cf0e-0755-4601-b643-a2e472e50445">
After change to 30vw:
<img width="833" alt="Screenshot 2024-08-23 at 12 35 13 PM" src="https://github.com/user-attachments/assets/0eb00c73-6fbe-41bc-a536-57c4000a6d75">